### DISCUSSION TITLE
Question: Integration with Microsoft Test Platform (Protocol) MTP(P)?

### DISCUSSION THREAD

**Discussion by turbobobbytraykov** [nunit/nunit#4838] [General]
Not sure if this an idea or a question.
Saw TUnit recently and that it builds upon the now-in-preview Microsoft Test Platform (Protocol) and I also saw this commit:
https://github.com/xunit/xunit/commit/729e406e19865adf700b00531ad1ea92055401dd#diff-5c1900dca99df01c3ddc3a884282164b04deee76e79b176e2d34cfa47f3b8907R102
So xunit are gearing up to have the option to use the MTP runner (and other underlying moving parts?).

Are there any plans for nUnit as well?

**Comment by OsirisTerje** [comment]
Yes, it is.

See https://github.com/nunit/nunit3-vs-adapter/issues/1152  

There also is a beta out on myget.org

**Reply by stevenaw** [reply]
@OsirisTerje @SeanKilleen I can't recall, but is this something we have documented much yet?

**Reply by OsirisTerje** [reply]
No, just the links in the issue there.  

I've been planning to get a formal beta out, or even a release of the adapter and document these things too, but I haven't had time.

**Reply by stevenaw** [reply]
Ok thanks @OsirisTerje . Quite understandable

**Reply by turbobobbytraykov** [reply]
Thank you for the details, @OsirisTerje ! At this point, we'll wait for a formal public beta & some documentation that we can try out.
